.idea/misc.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (Introduction-to-Machine-Learning)" project-jdk-type="Python SDK" />
```

Comment:
The project-jdk-name attribute should have a valid Python SDK name instead of 'Python 3.9 (Introduction-to-Machine-Learning)'